sass/components/rtl_support/_grid.scss

```diff
@@ -0,0 +1,93 @@
+// RTL Grid
+.row.rtl, .rtl .row {
+	.col {
+		float: right;
+		$i: 1;
+		@while $i <= $num-cols {
+			$perc: unquote((100 / ($num-cols / $i)) + "%");
+
+			&.s#{$i} {
+				margin-right: auto;
+			}
+			$i: $i + 1;
+		}
+		$i: 1;
+		@while $i <= $num-cols {
+			$perc: unquote((100 / ($num-cols / $i)) + "%");
+
+			&.offset-s#{$i} {
+				margin-right: $perc;
+			}
+
+			&.pull-s#{$i} {
+				left: $perc;
+			}
+
+			&.push-s#{$i} {
+				right: $perc;
+			}
+			$i: $i + 1;
+		}
+		@media #{$medium-and-up} {
+			$i: 1;
+			@while $i <= $num-cols {
+				$perc: unquote((100 / ($num-cols / $i)) + "%");
+
+				&.m#{$i} {
+					margin-right: auto;
+				}
+				$i: $i + 1;
+			}
+			$i: 1;
+			@while $i <= $num-cols {
+				$perc: unquote((100 / ($num-cols / $i)) + "%");
+
+				&.offset-m#{$i} {
+					margin-right: $perc;
+				}
+
+				&.pull-m#{$i} {
+					left: $perc;
+				}
+
+				&.push-m#{$i} {
+					right: $perc;
+				}
+				$i: $i + 1;
+			}
+		}
+		@media #{$large-and-up} {
+			$i: 1;
+			@while $i <= $num-cols {
+				$perc: unquote((100 / ($num-cols / $i)) + "%");
+
+				&.l#{$i} {
+					margin-right: auto;
+				}
+				$i: $i + 1;
+			}
+			$i: 1;
+			@while $i <= $num-cols {
+				$perc: unquote((100 / ($num-cols / $i)) + "%");
+
+				&.offset-l#{$i} {
+					margin-right: $perc;
+				}
+
+				&.pull-l#{$i} {
+					left: $perc;
+				}
+
+				&.push-l#{$i} {
+					right: $perc;
+				}
+				$i: $i + 1;
+			}
+		}
+	}
+	&.reverse > {
```

Comment:
@tomscholz @NonameSLdev 
This class is added in case you want to change the floating behavior of the .cols inside a .row.
eg, You have RTL content inside a grid but you want to pull columns to left (keeping the deeper tags responding to the parent's .rtl class and prevent adding extra .rtl class to nested tags).